packages/block-library/src/image/edit.js

```diff
@@ -58,7 +58,9 @@ const LINK_DESTINATION_CUSTOM = 'custom';
 const ALLOWED_MEDIA_TYPES = [ 'image' ];
 
 export const pickRelevantMediaFiles = ( image ) => {
-	return pick( image, [ 'alt', 'id', 'link', 'url', 'caption' ] );
+	return {
+		...pick( image, [ 'alt', 'id', 'link', 'url', 'caption' ] ),
```

Comment:
Why spread.

Reply:
I actually reverted this change prior to the nasty rebase attempt. The spread made sense in a prior iteration where I was changing the `alt` value. Thanks for calling out. :)